src/schema/submissions.ts

```diff
@@ -160,15 +160,21 @@ export const resolvers: IResolvers<unknown, Context> = traceResolvers({
         };
       }
 
-      const existingPost = await getPostByUrl(cleanUrl, ctx, info);
+      const existingPost = await ctx
+        .getRepository(ArticlePost)
```

Comment:
I think we should keep querying Post here

Reply:
It will not return the existing deleted post anymore since the Post entity in the graphorm will be going to use the new View entity, so it would think it was never in our DB.

Reply:
Right. I think we will have to make changes to the `Post` entity to include the `url` and `canonicalUrl` since it is currently available on `ArticlePost` only. Let me update those quickly.

Reply:
I think that is what we have here, we are querying directly to the DB by using the typeorm. That's why we had to introduce the change.